main.go

```diff
@@ -370,6 +377,17 @@ func main() {
 		}
 
 		os.Exit(task(token))
+	} else if cmd == "web" {
+		if len(remainingArgs) > 0 {
```

Comment:
The "remainingArgs" logic was added for other commands in order to maintain backwards compatibility with an old version of oauth2l that didn't have named args. Since "web" is a new feature, there is no obligation to accept remainingArgs as parameters, so you can switch this to check for opts.Web.Stop directly.

Reply:
Fixed! I also changed opts.Web.Stop from being a string to a boolean